user/advanced-configuration/managing-vm-kernel.md

```diff
@@ -318,13 +307,55 @@ You can safely ignore this error message:
 grub2-probe: error: cannot find a GRUB drive for /dev/mapper/dmroot. Check your device.map
 ~~~
 
+You may want to adjust some settings in `/etc/default/grub` (or better `/etc/default/grub.d`). For example, lower `GRUB_TIMEOUT` to speed up VM startup. You need to re-run `sudo update-grub` after making grub confugration changes.
+
 Then shutdown the VM.
 
+Go to Qubes VM Manger -> right click on the VM -> Qube settings -> Advanced -> choose `pvgrub2-phv` -> OK
```

Comment:
First of all - a typo: `grub2-pvh`. But also, (unfortunately) exact value here currently depends on the chosen virt mode:
- PV -> `pvgrub2`
- PVH -> `pvgrub2-pvh`
- HVM -> empty